drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -104,7 +104,7 @@ static void nrf5_get_eui64(uint8_t *mac)
 				   sizeof(factoryAddress));
 #endif
 	if (ret != 0) {
-		LOG_ERR("Unable to read EUI64 from the secure zone.")
+		LOG_ERR("Unable to read EUI64 from the secure zone.");
```

Comment:
Please add a note in the commit log saying this should be squashed into 0ab7df74b22.